src/service/riskScoring.js

```diff
@@ -0,0 +1,35 @@
+import Fingerprint from 'fingerprintjs2';
+import request from '@/class/singleton/request';
+
+/**
+ * @param {object[]} entries
+ * @returns {void}
+ */
+const sendFingerprint = () => {
+  const finger = new Fingerprint({
+    excludeCanvas: true,
+  });
+
+  finger.get(async (hash, entries) => {
+    /**
+     * @type {{
+     *   [key: string]: string|number,
+     * }}
+     */
+    const json = entries.reduce((accumulator, entry) => {
+      Object.assign(accumulator, {
```

Comment:
You can just return this expression

Reply:
eslint issue without assign
we can't change arguments